framework/xds_flags.py

```diff
@@ -190,6 +190,12 @@
     help="Whether to enable GFE debug headers and what value to use.",
 )
 
+ADD_DUALSTACK_SUPPORT = flags.DEFINE_bool(
+    "add_dualstack_support",
+    default=False,
+    help=(f"Add support for Dual Stack resources to the framework."),
```

Comment:
[Minor] We don't need `f` for a fixed string.

Reply:
Done